### PR TITLE
Set the  reply-to for transcript emails

### PR DESCRIPTION
**Pull request by mfairchild365**
The reply-to should be the site email for the site that the conversation was started on

### CODE REVIEW

File: src/UNL/VisitorChat/Conversation/Email.php [hunk 1]
```diff
@@ -99,7 +99,44 @@ public function isUrgent($isUrgent = true)
...
+        return $email;
+    }
+    
+    public function getFallBackSiteEmail()
+    {
+        $email = [];
+        
+        foreach (\UNL\VisitorChat\Controller::$fallbackURLs as $url) {
+            $sites = \UNL\VisitorChat\Controller::$registryService->getSitesByURL($url);
+            $site  = $sites->current();
```
Comment (kabel) [inline src/UNL/VisitorChat/Conversation/Email.php]: Used var.

**Comment by mfairchild365** [comment]
@kabel ready for another review

File: src/UNL/VisitorChat/Conversation/ConfirmationEmail.php [hunk 2]
```diff
@@ -23,8 +23,29 @@ public static function sendConversation(\UNL\VisitorChat\Conversation\Record $co
         
         $email = new $class($conversation, $to, $fromId, $options);
         
+        $email->setReplyTo($email->getReplyTo());
```
Comment (kabel) [inline src/UNL/VisitorChat/Conversation/ConfirmationEmail.php]: This seems a bit awkward in the traditional sense of a getter and setter--the getter doesn't actually relate to the setter. Other than that, it seems okay.

Comment (mfairchild365) [inline src/UNL/VisitorChat/Conversation/ConfirmationEmail.php]: Valid point

**Pull request merged**